src/prommis/uky/uky_flowsheet.py

```diff
@@ -861,5 +873,839 @@ def display_results(m):
+        costing_method_arguments={
+            "cost_accounts": CSX_mixer_settler_accounts,
+            "scaled_param": m.fs.CSX_mixer_settler.volume,
+            "source": 1,
+            "n_equip": 6,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # Precipitation costs
+    # 9.2 is UKy Rare Earth Element Precipitation - Polyethylene Tanks
+    reep_pe_tanks_accounts = ["9.2"]
+    m.fs.reep_pe_tanks = UnitModelBlock()
+    m.fs.reep_pe_tanks.capacity = Var(initialize=15.04, units=units.gal)
+    m.fs.reep_pe_tanks.capacity.fix()
+    m.fs.reep_pe_tanks.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": reep_pe_tanks_accounts,
+            "scaled_param": m.fs.reep_pe_tanks.capacity,
+            "source": 1,
+            "n_equip": 1,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # 9.3 is UKy Rare Earth Element Precipitation - Tank Mixer
+    reep_tank_mixer_accounts = ["9.3"]
+    m.fs.reep_tank_mixers = UnitModelBlock()
+    m.fs.reep_tank_mixers.power = Var(initialize=0.61, units=units.hp)
+    m.fs.reep_tank_mixers.power.fix()
+    m.fs.reep_tank_mixers.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": reep_tank_mixer_accounts,
+            "scaled_param": m.fs.reep_tank_mixers.power,
+            "source": 1,
+            "n_equip": 1,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # 9.4 is UKy Rare Earth Element Precipitation - Process Pump
+    reep_pump_accounts = ["9.4"]
+    m.fs.reep_pump = UnitModelBlock()
+    flow_9_4 = value(
+        units.convert(
+            flowsheet.fs.precipitator.aqueous_inlet.flow_vol[0],
+            to_units=units.gal / units.min,
+        )
+    )
+    m.fs.reep_pump.feed_rate = Var(initialize=flow_9_4, units=units.gal / units.min)
+    m.fs.reep_pump.feed_rate.fix()
+    m.fs.reep_pump.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": reep_pump_accounts,
+            "scaled_param": m.fs.reep_pump.feed_rate,
+            "source": 1,
+            "n_equip": 1,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # 9.5 is UKy Rare Earth Element Precipitation - Filter Press
+    reep_filter_press_accounts = ["9.5"]
+    m.fs.reep_filter_press = UnitModelBlock()
+    m.fs.reep_filter_press.volume = Var(initialize=0.405, units=units.ft**3)
```

Comment:
Same response here - I don't have a strong intuitive sense of what a filter press volume should be, but let me know if there's a value I should use instead.

Reply:
This would correspond to the feedrate and kinetics of the precipitation tank. I have actually typically seen filter presses sized to the m**2. I think this assumption is ok for now (it seems reasonable for a bench or pilot scale).